src/plugincollection.js

```diff
@@ -293,6 +298,29 @@ export default class PluginCollection {
 		}
 	}
 
+	/**
+	 * Runs the initialisation process ({@link module:core/plugin~Plugin#init `init()`}
+	 * and {@link module:core/plugin~Plugin#afterInit `afterInit()`} methods) for the given set of plugins.
+	 *
+	 * @param {Array.<module:core/plugin~PluginInterface>} plugins The array of plugins to initialise.
+	 * @returns {Promise} A promise which resolves after all given plugins have been initialized.
+	 */
+	init( plugins ) {
+		return initPlugins( plugins, 'init' )
+			.then( () => initPlugins( plugins, 'afterInit' ) )
+			.then( () => { this.fire( 'ready' ); } );
```

Comment:
We agreed that since we have no real use cases for this event we should remove it.